api_tests/src/image.spec.ts

```diff
@@ -207,7 +207,7 @@ test("Images in remote image post are proxied if setting enabled", async () => {
   ).toBeTruthy();
 
   // Make sure that it ends with jpg, to be sure its an image
```

Comment:
it's the same on main, but these comments no longer describe the check that is performed.

Reply:
Fixed, also pushing that to main branch.